pom.xml

```diff
@@ -95,13 +96,15 @@ limitations under the License.
     <version.lib.geopackage>6.6.4</version.lib.geopackage>
     <version.lib.graalvm>23.0.3</version.lib.graalvm>
     <version.lib.guava>33.0.0-jre</version.lib.guava>
+    <version.lib.hadoop>3.4.0</version.lib.hadoop>
     <version.lib.hikari>5.1.0</version.lib.hikari>
     <version.lib.ipresource>1.52</version.lib.ipresource>
     <version.lib.jackson>2.16.1</version.lib.jackson>
     <version.lib.jts>1.19.0</version.lib.jts>
     <version.lib.junit>5.10.2</version.lib.junit>
     <version.lib.log4j>3.0.0-beta2</version.lib.log4j>
     <version.lib.lucene>9.10.0</version.lib.lucene>
+    <version.lib.parquet>1.13.1</version.lib.parquet>
```

Comment:
Version 1.14.0 was release on Maven central I believe. Do you think we should already use it?

Reply:
Yes, we should probably upgrade but it raises serialization issues.

Reply:
Yes, it looks like it's missing some serializer. Maybe it has moved to another module.

```
shaded.parquet.com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class org.apache.parquet.schema.LogicalTypeAnnotation$StringLogicalTypeAnnotation
```